notebook.py

```
@@ -0,0 +1,53 @@
+# MAGIC %pip install pyyaml
+# MAGIC import yaml
+
+# COMMAND ----------
+
+from scr.nyctaxi.data_processor import DataProcessor
+
+# COMMAND ----------
+
+# Load configuration
+with open("project_config.yml", "r") as file:
+    config = yaml.safe_load(file)
```

Ruff

Undefined name `yaml`

(F821)